venvs/tests/test_make.py

```diff
@@ -140,6 +140,44 @@ def test_multiple_installs_one_link(self):
     def test_multiple_installs_one_link_no_name(self):
         self.run_cli(["-i", "foo", "-i", "bar", "-l", "foo"], exit_status=2)
 
+    def test_install_edit_config(self):
+        """Install will automatically edit the config file."""
```

Comment:
Minor: can you use present tense for these, i.e. `Install automatically edits the config file.` (though that's not true anymore I guess, it's only when the option is provided).